benchmarks/overrides_benchmark/README.md

```diff
@@ -0,0 +1,19 @@
+# `__torch_function__` micro-benchmarks
+
+This benchmark suite provides a systemic way to measure the performance of `__torch_function__` overhead.
+
+## Getting started
+### Initial Setup
+Install `py-spy` by doing:
+
+```bash
+pip install py-spy
+```
+
+### Running the benchmark
+Run the following commands in the terminal, with the working directory being `${PYTORCH_CLONE_DIR}/benchmarks/overrides_benchmark`:
+
+```bash
+python bench.py
+py-spy record -o profile.svg --native -- python pyspybench.py
```

Comment:
let's change this to have profiles for each of the 3 relevant cases, on master and on the feature branch